osu.Game/Configuration/OsuConfigManager.cs

```diff
@@ -18,7 +18,6 @@ protected override void InitialiseDefaults()
             Set(OsuConfig.MouseSpeed, 1.0);
 
             Set(OsuConfig.Username, string.Empty);
-            Set(OsuConfig.Password, string.Empty);

```

Comment:
But we don't use password filed anywhere, so why we still need it at all? Doesn't it pointless to keep line in the file, which will be empty all the time?